docs/sources/alerting/_index.md

```diff
@@ -2,39 +2,71 @@
 aliases:
   - /docs/grafana/latest/alerting/
   - /docs/grafana/latest/alerting/unified-alerting/alerting/
+  - /docs/grafana/latest/about-alerting
 title: Alerting
 weight: 114
 ---
 
 # Grafana Alerting
 
-Grafana Alerting allows you to learn about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services. It centralizes alerting information in a single, searchable view that allows you to:
+Grafana Alerting allows you to learn about problems in your systems moments after they occur. Create, manage, and take action on your alerts in a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.
```

Comment:
Maybe worth adding an additional sentence to underline the important of the "any datasource"?

```suggestion
Grafana Alerting allows you to learn about problems in your systems moments after they occur. Create, manage, and take action on your alerts in a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.

Grafana Alerting works with queries and expressions from any existing data source — no matter where your data is stored.
```